nightfall-optimist/src/routes/proposer.mjs

```diff
@@ -305,6 +312,12 @@ router.get('/mempool', async (req, res, next) => {
   }
 });
 
+/**
+ * Function to Propose a state update block  This just
+ * provides the tx data, the user will need to call the blockchain client
+ * @deprecated - this is now an automated process - no need to manually propose
+ * a block
+ */
```

Comment:
FYI

I removed this one myself. It says "deprecated" but it is not true since the adversary (or at least the old one) uses this endpoint (see test util `createBadBlock`).

At the same time, it will clash with the Swagger so...

Reply:
what do i do @imagobea ?

Reply:
IMO we can live without it

Reply:
so, i just remove the `deprecated` or should i do something else @imagobea ?

Reply:
Sorry I missed your latest comment @druiz0992 
I would remove the entire docstring from L315-320, thank you (remember it will be properly documented when we have the swagger). Thanks again!